insights/parsers/etc_machine_id.py

```diff
@@ -24,10 +25,13 @@ class EtcMachineId(Parser):
         <class 'insights.parsers.etc_machine_id.EtcMachineId'>
         >>> machine_id.machine_id
         '4f3fbfda59ff4aea969255a73342d439'
+        >>> machine_id.uuid
+        '4f3fbfda-59ff-4aea-9692-55a73342d439'
     """
 
     def parse_content(self, content):
         content = get_active_lines(content)
         if not content or len(content) != 1:
             raise SkipComponent('Invalid Content')
-        self.machine_id = content[0]
+        self.id = self.machine_id = content[0]
+        self.uuid = str(uuid.UUID(self.id))
```

Comment:
`uuid.UUID()` could throw error when failed: 
```
raise ValueError('badly formed hexadecimal UUID string')
```

Should we give it an exception handling of this `uuid.UUID()` validation?

Reply:
Same for `insights/parsers/client_metadata.py` and `insights/parsers/subscription_manager.py` .

Reply:
The "ValueError" will also result in the parser failure, just like the "ContentException", I think it's a bit of "draw legs on a snake" if we catch the "valueError" here and re-raise it with "ContentException.